src/Infrastructure/Infrastructure.csproj

```diff
@@ -20,4 +18,4 @@
 	<ItemGroup>
 		<Folder Include="Data\Migrations\" />
 	</ItemGroup>
```

Comment:
This one is needed, but I can't get why? AFAIK, csproj always references *\*.cs* files, and all files in Data\Migrations\ are .cs.

Why the build fails when it's deleted